src/components/header/PrimaryNav/PrimaryNav.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { PrimaryNav } from './PrimaryNav'
+
+export default {
+  title: 'PrimaryNav',
+  parameters: {
+    info: `
+USWDS 2.0 PrimaryNav component
+
+Source: https://designsystem.digital.gov/components/header/
+`,
+  },
+}
+
+const testBasicHeaderItems = [
+  <a className="usa-current" href="#linkOne" key="one">
```

Comment:
I wonder if we should be more flexible here?  🤔  I assumed that instead of defining `<a>` tags we would accept any navigation link components that the consumer of our library passes in.  Thinking specifically of apps that use [react-router](https://reacttraining.com/react-router/web/api/NavLink).